macros/contexts/contextInteger.pl

```diff
@@ -235,8 +235,8 @@ sub isPrime {
 sub randomPrime {
 	my ($start, $end) = @_;
 	my @primes = context::Integer::_getPrimesInRange($start, $end);
-	$self->Error("Could not find any prime numbers in range.") if $#primes == 0;
-	my $primeIndex = $main::PG_random_generator->random(0, ($#primes - 1), 1);
+	Value::Error('Could not find any prime numbers in range.') if @primes == 0;
+	my $primeIndex = main::random(0, $#primes);
 	return $primes[$primeIndex];
```

Comment:
Lines 239 and 240 could be replaced with:
```
return main::list_random(@primes);

Reply:
That is also better.  Thanks.